cmd/operator/update.go

```
@@ -43,6 +45,33 @@ func NewCmdUpdate() *cobra.Command {
 		Aliases: []string{"opr"},
 		Short:   "Update core operator",
 		PreRunE: func(cmd *cobra.Command, args []string) error {
+
+			operatorIndex, err := index.NewOperator()
+			if err != nil {
+				return err
+			}
+
+			if !force {
+				laststr, _ := operatorIndex.Last(cmd.Context())
+				last, _ := semver.NewSemver(laststr)
```

Please never ignore errors.